package.json

```diff
@@ -14,12 +22,15 @@
     "node": ">=16.0.0"
   },
   "scripts": {
-    "dev": "node --experimental-json-modules ./node_modules/.bin/rollup --config rollup.config.js -w",
-    "build": "tsup --dts",
-    "docs": "typedoc --options typedoc.json --plugin typedoc-plugin-markdown  --out docs src/index.ts ",
+    "dev": "concurrently \"bun run dev:rollup\" \"bun run dev:types\"",
```

Comment:
I've changed the dev command to concurrently run rollup and tsup, so types and files and updated in real time during development. Both will be running in watch mode until closed.